locale/sv/LC_MESSAGES/django.po

```diff
@@ -522,6 +522,12 @@ msgstr "Felaktig pk \"{pk_value}\" - objektet existerar inte."
 msgid "Invalid value in filter %(filter)s"
 msgstr "Felaktigt värde i filter %(filter)s"
 
+msgid "You need to login with strong authentication to reserve this resource."
+msgstr ""
```

Comment:
"Du måste logga in med en stark autentiseringsmetod för att reservera den här resursen"